sdk/python/kfp/__init__.py

```diff
@@ -16,7 +16,10 @@
 # https://packaging.python.org/guides/packaging-namespace-packages/#pkgutil-style-namespace-packages
 __path__ = __import__('pkgutil').extend_path(__path__, __name__)
 
-__version__ = '2.13.0'
```

Comment:
can you also update the release docs (step 2): https://github.com/kubeflow/pipelines/blob/8a402c10a8e3d36f964fb451760319c99a185e8f/RELEASE.md#releasing-kfp-python-packages